script/entrypoint

```diff
@@ -33,6 +33,12 @@ elif [ "$1" = "query" ]; then
     # For queries with public_json set in the metadata, the results will be
     # exported as JSON to Cloud storage.
     python script/publish_public_data_json --query_file="${@: -1}" ${@:1:$#-1}
+elif [ "$1" = "publish_gcs_metadata" ]; then
+    # For an invocation like:
+    #     publish_gcs_metadata [options]
+    # we dispatch to a script that publishes metadata files to GCS for datasets
+    # that have previsously been made public on GCS.
+    exec script/publish_public_data_gcs_metadata ${@:2}
```

Comment:
This will be needed for Airflow

Reply:
We can call the full path to the script (`script/publish_public_data_gcs_metadata`) from Airflow. That's what we do for copy_deduplicate, and I think we should stick to that rather than adding additional logic here for new cases.